Fix L2 nightly test_layernorm.py and a few other tests

tests/ttnn/nightly/unit_tests/operations/fused/test_layernorm.py is failing on main:
https://github.com/tenstorrent/tt-metal/actions/runs/26388857253/job/77708151873#step:6:40
```
tests/ttnn/nightly/unit_tests/operations/fused/test_layernorm.py:13: in <module>
    from models.common.utility_functions import pad_by_zero, torch2tt_tensor, run_for_blackhole
E   ImportError: cannot import name 'pad_by_zero' from 'models.common.utility_functions' (/work/models/common/utility_functions.py)
```

Looks like https://github.com/tenstorrent/tt-metal/pull/43958 missed some imports.

Need to update relevant tests.

--- a/tests/tt_eager/python_api_testing/sweep_tests/pytests/tt_dnn/test_softmax_sharded.py
+++ b/tests/tt_eager/python_api_testing/sweep_tests/pytests/tt_dnn/test_softmax_sharded.py
@@ -17,7 +17,7 @@ from tt_lib.utils import (
     is_close,
 )
 from models.common.utility_functions import print_diff_argmax, comp_pcc
-from models.common.utility_functions import torch2tt_tensor, tt2torch_tensor
+from models.common.utility_functions import torch2tt_tensor
 from models.common.utility_functions import is_wormhole_b0, is_blackhole
 
 # only use certain tests for CI to reduce run time

--- a/tests/tt_eager/python_api_testing/unit_testing/misc/test_update_cache.py
+++ b/tests/tt_eager/python_api_testing/unit_testing/misc/test_update_cache.py
@@ -6,7 +6,6 @@ import torch
 import pytest
 import ttnn
 from loguru import logger
-from models.common.utility_functions import nearest_32
 from tests.tt_eager.python_api_testing.sweep_tests.comparison_funcs import comp_pcc, comp_equal
 from models.common.utility_functions import skip_for_blackhole
 

--- a/tests/ttnn/nightly/unit_tests/operations/fused/test_layernorm.py
+++ b/tests/ttnn/nightly/unit_tests/operations/fused/test_layernorm.py
@@ -10,7 +10,7 @@ import torch
 import ttnn
 
 
-from models.common.utility_functions import pad_by_zero, torch2tt_tensor, run_for_blackhole
+from models.common.utility_functions import torch2tt_tensor, run_for_blackhole
 from tests.ttnn.utils_for_testing import assert_numeric_metrics
 
 TEST_PADDING_VALUE = -42

--- a/tests/ttnn/nightly/unit_tests/operations/fused/test_softmax_interleaved.py
+++ b/tests/ttnn/nightly/unit_tests/operations/fused/test_softmax_interleaved.py
@@ -16,7 +16,6 @@ from tt_lib.utils import (
 )
 from models.common.utility_functions import print_diff_argmax
 from tests.ttnn.utils_for_testing import assert_numeric_metrics
-from models.common.utility_functions import torch2tt_tensor
 
 TEST_PADDING_VALUE = -42
 

--- a/tests/ttnn/nightly/unit_tests/operations/fused/test_softmax_sharded.py
+++ b/tests/ttnn/nightly/unit_tests/operations/fused/test_softmax_sharded.py
@@ -14,7 +14,6 @@ from tt_lib.utils import (
     tilize_to_list,
     untilize,
 )
-from models.common.utility_functions import torch2tt_tensor
 from tests.ttnn.utils_for_testing import assert_numeric_metrics
 
 

--- a/tests/ttnn/nightly/unit_tests/operations/matmul/test_rs_matmul_1d_gather_in0.py
+++ b/tests/ttnn/nightly/unit_tests/operations/matmul/test_rs_matmul_1d_gather_in0.py
@@ -5,7 +5,7 @@
 import pytest
 from loguru import logger
 import ttnn
-from models.common.utility_functions import is_wormhole_b0, is_blackhole
+from models.common.utility_functions import is_blackhole
 import torch
 import itertools
 import os

--- a/tests/ttnn/nightly/unit_tests/operations/matmul/test_rs_matmul_1d_gather_in0.py
+++ b/tests/ttnn/nightly/unit_tests/operations/matmul/test_rs_matmul_1d_gather_in0.py
@@ -6,7 +6,6 @@ import pytest
 from loguru import logger
 import ttnn
 from models.common.utility_functions import is_wormhole_b0, is_blackhole
-from models.common.utility_functions import torch2tt_tensor, roundup32
 import torch
 import itertools
 import os

